src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Storage {
+    private String filepath;
+    Storage(String filepath) {
+        this.filepath = filepath;
+    }
+
+    public void save(String saveString) throws LukeException {
+        try {
+            FileWriter fw = new FileWriter(filepath);
+            fw.write(saveString);
+            fw.close();
+        } catch (IOException e) {
+            throw new LukeException("Error writing saved tasks into '" + filepath
+                    + "'\n\n Save aborted");
+        }
+    }
+
+    public ArrayList<Task> load() throws LukeException {
```

Comment:
Perhaps this function could have a more descriptive name? For example loadFromTaskFile